develop-guide/writeblitz/writeblitz.rst

```diff
@@ -550,6 +569,41 @@ Action ``compare`` allows you to verify the values of the saved variables. Below
         - "'%VARIABLES{os}' == 'NX-OS' and '%VARIABLES{date_created}' == '10/22/2019 10:00:00 [10/22/2019 16:57:31]'"
         - " %VARIABLES{bootflash} >= 290000 or '%VARIABLES{bios}' == '07.33'"
 
+
+
+.. note::
```

Comment:
why is this note in compare?

Reply:
Its not in compare its at the end the list of actions

Reply:
What about creating a section called Tips and put all of them there

Reply:
like all the notes?